graphql_compiler/compiler/cypher_query.py

```diff
@@ -47,7 +48,9 @@
 
 def _get_all_supertypes_of_exact_type(query_metadata_table, exact_type):
     """Return the set of all supertypes of the given exact type."""
-    # TODO(predrag): Plumb the SchemaGraph work through here.
+    # TODO(predrag): Plumb the SchemaGraph work through here. When/ if this implementation changes,
+    # we may also need to modify how CypherStep objects for fold directives are constructed in
+    # _generate_CypherStep_list_from_ir_blocks based on the comment there.
```

Comment:
`_generate_CypherStep_list_from_ir_blocks` seems like it was the victim of a find-and-replace, since the capitalization doesn't match the function's actual name